build/common.go

```diff
@@ -0,0 +1,286 @@
+//+build mage
+
```

Comment:
It turns out Mage can't import packages with the "mage" build tag, it results in an error in the parse stage. I received confirmation from the author that they won't support the build tag either, for backwards compatibility reasons.
```suggestion
```